c7n/resources/awslambda.py

```diff
@@ -394,6 +394,31 @@ def process(self, functions):
         self.log.debug("Deleted %d functions", len(functions))
 
 
+@AWSLambda.action_registry.register('modify-security-groups')
+class LambdaModifyVpcSecurityGroups(ModifyVpcSecurityGroupsAction):
+
+    permissions = ("lambda:UpdateFunctionConfiguration",)
+
+    def process(self, functions):
+        client = local_session(self.manager.session_factory).client('lambda')
+        groups = super(LambdaModifyVpcSecurityGroups, self).get_groups(
+            functions)
+        # should possibly return only VPC-enabled lambdas here?
+
+        for idx, i in enumerate(functions):
+            if 'VpcConfig' in i:
```

Comment:
It’s nicer to continue if this is not the case so you can decent more of the code here. Ie if not vpcconfig in i: continue

Reply:
ended up wrapping it in a try/except instead--a KeyError means it's not in a vpc

Reply:
so we generally don't use exceptions handlers for common paths, ie we use them for exceptions. Simply checking if the key exists and has a value is preferrable and simplifies the code path, and helps achieve the purpose of the original comment which was to dedent this block (ie. avoid an extra level of nesting).

Reply:
thanks for clarifying, I switched it back to your initial suggestion and wrote a simple test for it.